### PR TITLE
feat: added get paged exchange-accounts

### PR DESCRIPTION
**Pull request by AmitShe**

### CODE REVIEW

File: fireblocks_sdk/api_types.py [hunk 1]
```diff
@@ -257,6 +257,24 @@ def __init__(self, name_prefix=None, name_suffix=None, min_amount_threshold=None
...
+    """ Optional filters to apply for request
+
+    Args
+
+        limit (number, optional): Results page size
+        before (string, optional): cursor string received from previous request
+        after (string, optional): cursor string received from previous request
+
+    Constraints
+        - You should only insert 'before' or 'after' (or none of them), but not both
```
Comment (YoavBZ) [inline fireblocks_sdk/api_types.py:270]: Instead of relying on the docstring, please add the relevant check inside the constructor

**Review by YoavBZ — APPROVED** [review]

**Review by YoavBZ — APPROVED** [review]

**Comment by YoavBZ** [comment]
Please resolve the conflicts

File: fireblocks_sdk/sdk.py [hunk 2]
```diff
@@ -726,6 +727,35 @@ def get_exchange_accounts(self):
 
         return self._get_request("/v1/exchange_accounts")
 
+    def get_exchange_accounts_paged(self, paged_exchange_accounts_request_filters: PagedExchangeAccountRequestFilters=PagedExchangeAccountRequestFilters()):
```
Comment (YoavBZ) [inline fireblocks_sdk/sdk.py:730]: Mutable default arguments should be avoided. Please use `None` and add an if-statement inside the method

Comment (AmitShe) [reply fireblocks_sdk/sdk.py]: remove it (should behave like 'get_vault_accounts_with_page_info')

**Comment by NoamRodrik** [comment]
Approved :) @YoavBZ You're good with it?

**Pull request merged**